techlibs/common/synth.cc

```diff
@@ -17,17 +17,16 @@
  *
  */
 
-#include "kernel/register.h"
```

Comment:
There are unrelated whitespace changes but as we discussed in this week's Yosys dev call we can live with it and take it up including those. I went over this file and the only non-formatting change is the addition of the `-booth` flag